### PR TITLE
ISPN-9775 - ISPN build for IBM Java ends with error in Ant execution task

### PR DESCRIPTION
**Pull request by andyuk1986**

### CODE REVIEW

File: server/hotrod/pom.xml [hunk 1]
```diff
@@ -153,6 +153,10 @@
                               <arg value="${project.build.testOutputDirectory}/password_client_truststore.p12"/>
                               <arg value="-deststorepass"/>
                               <arg value="secret"/>
+                              <arg value="-srcstoretype"/>
+                              <arg value="pkcs12"/>
+                              <arg value="-deststoretype"/>
+                              <arg value="pkcs12"/>
```
Comment (wburns) [inline server/hotrod/pom.xml]: Do we not want "JCEKS" as the other one in client hotrod?

Comment (andyuk1986) [reply server/hotrod/pom.xml]: @wburns I don't know. According to the last commits in server/hotrod all keystores were changed to pkcs12. I have just added the required part for IBM JDK to be able to build them.

Comment (wburns) [reply server/hotrod/pom.xml]: Okay, it seems to be fine with our CI, but I haven't confirmed with IBM, so I trust you guys on that front :)

**Comment by wburns** [comment]
Integrated into master, thanks @andyuk1986

**Pull request merged**